pkgs/applications/networking/owncloud-client/libre-graph-api-cpp-qt-client.nix

```diff
@@ -19,17 +18,9 @@ stdenv.mkDerivation rec {
 
   sourceRoot = "${src.name}/client";
 
-  nativeBuildInputs = [ cmake wrapQtAppsHook ];
+  nativeBuildInputs = [ cmake ];
   buildInputs = [ qtbase ];
 
+  dontWrapQtApps = true;
```

Comment:
If there is no wrapper, so we even need to set this?

Reply:
I just tried, it seems not needed.